src/app/shared/launch.service.ts

```diff
@@ -27,4 +27,22 @@ export abstract class LaunchService {
         return `$ consonance run --tool-dockstore-id ${path}:${versionName} ` +
             '--run-descriptor Dockstore.json --flavour \<AWS instance-type\>';
     }
+
+    /**
+     * This creates the Dockstore-supported cwltool launch command
+     * @param path The GA4GH Tool's path
+     * @param versionName The ToolVersion's name
+     */
+    getDockstoreSupportedCwlLaunchString(path: string, versionName: string) {
+        return `$ cwltool --non-strict ${path}:${versionName} Dockstore.json`;
```

Comment:
I think this should be left out here. Users can add it back in if they wish

Reply:
the '--non-strict'?

Reply:
Correct, my apologies for the imprecision